scripts-available/CDB_Organizations.sql

```diff
@@ -25,6 +25,57 @@ BEGIN
 END
 $$ LANGUAGE PLPGSQL VOLATILE;
 
+-------------------------------------------------------------------------------
+-- Administrator
+-------------------------------------------------------------------------------
+CREATE OR REPLACE
+FUNCTION cartodb._CDB_Organization_Admin_Role_Name()
+    RETURNS TEXT
+AS 'SELECT ''cdb_org_admin''::text || ''_'' || md5(current_database());'
```

Comment:
Ok, without `md5` then.

I've added a check at `cartodb._CDB_Group_GroupRole`. This way we don't have to propagate error handling and all generated roles will be valid.